samples/tab-adaptive-cards/csharp/TabWithAdpativeCardFlow/AppPackage/manifest.json

```diff
@@ -0,0 +1,45 @@
+﻿{
+  "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.9",
+  "version": "1.0.0",
+  "id": "4d9e95ab-ebe3-4976-82e9-3526d5c54906",
+  "packageName": "com.microsoft.sequentialflowbot",
```

Comment:
Please change package name according to your application.

Reply:
changed